cmd/egress/cmd.go

```diff
@@ -93,17 +92,18 @@ are set correctly before execution.
 # Verify that essential OpenShift domains are reachable from a given SUBNET_ID/SECURITY_GROUP association
 ./osd-network-verifier egress --subnet-id ${SUBNET_ID} --security-group-ids ${SECURITY_GROUP}`,
 		Run: func(cmd *cobra.Command, args []string) {
-			// OSD-20380 - remapping for backwards compatibility
-			platformType, err := helpers.GetPlatformType(config.platformType)
+			platformType, err := platform.ByName(config.platformType)
 			if err != nil {
-				// Unknown platformType specified
-				fmt.Printf("unknown platform type '%v'\n", config.platformType)
+				//Unknown platformType specified
+				fmt.Println(err)
 				os.Exit(1)
 			}
 
+			normalizedPlatformType := platformType.String()
```

Comment:
Similarly to above, do we really need to "normalize" Platforms to string anymore? Can't we use Platforms directly?